x/merkledb/db.go

```diff
@@ -870,11 +870,6 @@ func (db *merkleDB) commitBatch(ops []database.BatchOp) error {
 	return view.commitToDB(context.Background())
 }
 
-// CommitToParent is a no-op for the db because it has no parent
-func (*merkleDB) CommitToParent(context.Context) error {
```

Comment:
This should've been removed in https://github.com/ava-labs/avalanchego/pull/1854. Woops.